js/apps/system/_admin/aardvark/APP/frontend/js/views/databaseView.js

```diff
@@ -1,513 +0,0 @@
-      var buttons = [];
-      var tableContent = [];
-
-      $.when(this.getDatabaseSettings(dbName)).done(function(response){
-        tableContent.push(
-          window.modalView.createReadOnlyEntry('id_name', 'Name', _.escape(dbName), '')
-        );
-  
-        tableContent.push(
-          window.modalView.createReadOnlyEntry('replication_factor', 'Replication factor', response.result.replicationFactor, '')
-        );
-  
-        tableContent.push(
-          window.modalView.createReadOnlyEntry('write_concern', 'Write concern', response.result.writeConcern, '')
-        );
-  
-        tableContent.push(
-          window.modalView.createReadOnlyEntry('sharding', 'Sharding', response.result.sharding, '')
-        );
-      });
-
-      if (isDeletable) {
-        buttons.push(
-          window.modalView.createDeleteButton(
-            'Delete',
-            this.submitDeleteDatabase.bind(this, dbName)
-          )
-        );
-      } else {
-        buttons.push(window.modalView.createDisabledButton('Delete'));
-      }
-      window.modalView.show(
-        'modalTable.ejs',
-        'Delete database',
-        buttons,
-        tableContent
-      );
-    },
-
-    createAddDatabaseModal: function () {
-      var self = this;
-      $.ajax({
-        type: 'GET',
-        cache: false,
-        url: arangoHelper.databaseUrl('/_admin/server/databaseDefaults'), //get default properties
-        contentType: 'application/json',
-        processData: false,
-        success: function (data) {
-          self.createAddDatabaseModalReal(data);
-        },
-        error: function () {
-          arangoHelper.arangoError('Engine', 'Could not fetch ArangoDB database defaults.');
-        }
-      });
-    },
-
-    createAddDatabaseModalReal: function (data) {
-      var dbDefaultProperties = data;
-      var buttons = [];
-      var tableContent = [];
-      var databaseNameValidations =
-        window.arangoValidationHelper.getDatabaseNameValidations();
-      // Database Name
-      tableContent.push(
-        window.modalView.createTextEntry(
-          'newDatabaseName',
-          'Name',
-          '',
-          false,
-          'Database Name',
-          true,
-          databaseNameValidations
-        )
-      );
-
-      if (window.App.isCluster) {
-        // (id, label, value, info, placeholder, mandatory, regexp) 
-        // ReplicationFactor
-        tableContent.push(
-          window.modalView.createTextEntry(
-            'new-replication-factor',
-            'Replication factor',
-            dbDefaultProperties.replicationFactor,
-            'Numeric value. Must be at least 1. Total number of copies of the data in the cluster',
-            'Default replication factor',
-            false,
-            [
-              {
-                rule: Joi.string().allow('').optional().regex(/^([1-9]|10)$/),
-                msg: 'Must be a number between 1 and 10.'
-              }
-            ]
```

Comment:
We are missing these validations in the React form, let's add them

Reply:
Just trying to avoid regressions. If you think this is okay, I'm good with it.

For reference - here's a comment by @jsteemann on a previous PR that it's currently hardcoded to max 10 on the backend

>The current hard-coded maximum value is 10, although the effective maximum value for replicationFactor is the minimum of the number of available DB servers and 10.
The effective maximum value for writeConcern is the effective maximum value of replicationFactor.
The current validation will allow entering much higher values, which will then be rejected by the server.

https://github.com/arangodb/arangodb/pull/19119#discussion_r1210181282

Reply:
As discussed in team meeting, we are not adding these old validations are they are practically undocumented